src/consts/models.tsx

```diff
@@ -5,6 +5,14 @@ import { KeyboardType, StyleProp, ViewStyle, Platform } from 'react-native';
 import { ButtonProps } from 'react-native-elements';
 
 import { FastImageSource } from 'app/components';
+import {
+  HDSegwitP2SHAirWallet,
+  HDSegwitP2SHArWallet,
+  HDSegwitBech32Wallet,
+  SegwitP2SHWallet,
+  HDSegwitP2SHWallet,
+} from 'app/legacy';
+import { ImportWalletType } from 'app/screens/ImportWalletChooseTypeScreen';
```

Comment:
I would add import `ImportWalletType` to consts/models and import it in the `ImportWalletChooseTypeScreen`

Reply:
Done